packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/errors.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+from models_library.errors_classes import OsparcErrorMixin
+
+
+class CatalogApiBaseError(OsparcErrorMixin, Exception):
+    def __init__(self, **ctx: Any) -> None:
+        super().__init__(**ctx)
+
```

Comment:
Q: why does it need one of these ugly python __init__ ? is this not a pydantic mixin?

Reply:
This is to enforce key-arguments and pass them as context. Note that Exception accepts positional arguments as well